src/code/z_player_lib.c

```diff
@@ -599,11 +614,17 @@ u8 sEyeMouthIndexes[][2] = {
     { 7, 2 }, { 0, 2 }, { 3, 0 }, { 4, 0 }, { 2, 2 }, { 1, 1 }, { 0, 2 }, { 0, 0 },
 };
 
-u8* sEyeTextures[] = {
+/*
+ *Link's eye and mouth textures are placed at the exact same place in adult and child Link's respective object files.
+ *This allows the array to only contain the symbols for one file and have it apply to both. This is a problem for
+ *shiftability, and changes will need to be made in the code to account for this in a modding scenario. The symbols from
+ *adult Link's object are used here.
+ */
```

Comment:
add a space after the star. think the formatter should have done this? if so means you should run it again

Reply:
i guess the formatter doesnt fix it, need to do it manually

Reply:
and we do a double star on the top one 
/**
*

doing this, vscode will automatically format it properly